python/core/auto_generated/qgsaggregatecalculator.sip.in

```diff
@@ -101,10 +101,20 @@ Sets a filter to limit the features used during the aggregate calculation.
     void setFidsFilter( const QgsFeatureIds &fids );
 %Docstring
 Sets a filter to limit the features used during the aggregate calculation.
-If an expression filter is set, it will override this filter.
+If an expression filter is set, it will override this filter unless filter stacking is enabled (sackFilters)
```

Comment:
Can you please split off these changes to aggregate calculator so that we can get the feature request changes reviewed independantly?

Reply:
Sure but I'll have to branch off the code from aggregate and duplicate the code from the feature request to get the function, otherwise i fear that I might forget to connect the things by the time everything gets merged.

Reply:
see #31847

Reply:
And I forgot that the tests relied on the aggregate calc and not only the feature request. Do you still want things split off and not working or I can merge them back?

Reply:
Please split :) The new test in the feature source test will fully cover all the feature-request specific additions, so you can leave off the other test additions to the aggregate specific follow up

Reply:
Let me know if this is what you had in mind about the split or if I misinterpreted your intentions.